hack/lib/etcd.sh

```diff
@@ -40,7 +40,7 @@ kube::etcd::validate() {
    export PATH=$KUBE_ROOT/third_party/etcd:$PATH
    hash etcd
    echo $PATH
-   ls $KUBE_ROOT/third_party/etcd
```

Comment:
Let's not mix up things here. Let's keep this PR strictly to typo fixes. You already have a PR for it anyway - PR #39016. So please remove this from here.